pygmt/datasets/load_earth_dataset.py

```diff
@@ -0,0 +1,210 @@
+"""
+Internal function to load GMT remote datasets.
+"""
+from typing import Dict, NamedTuple
+
+from pygmt.exceptions import GMTInvalidInput
+from pygmt.helpers import kwargs_to_strings
+from pygmt.io import load_dataarray
+from pygmt.src import grdcut, which
+
+
+class Resolution(NamedTuple):
+    """
+    The available grid registrations for a given resolution and whether it is a
+    tiled grid.
+
+    Parameters
```

Comment:
We should use `Attributes` for class attributes.
```suggestion
    Attributes
```